vignettes/GeomxTools_RNA-NGS_Analysis.Rmd

```diff
@@ -113,6 +118,26 @@ library(GeomxTools)
 library(GeoMxWorkflows)
 ```
 
+```{r checking package versions}
```

Comment:
Can you email this section to Prajan with a copy of the vignette to show him what the vignette currently looks like, cc me? He needs to know when we add sections. Maybe even give him a link to this diff or just take a snapshot of the highlights to add to the email.

Reply:
These checks are incorrect. calling `vignettes(GeoMxWorkflow)` when the geomxworkflow package is updated will give you the same advanced vignette